thoughts-of-a-rustacean-learning-go.md

```diff
@@ -0,0 +1,244 @@
+<!--
+headline: Thoughts of a Rustacean learning Go
```

Comment:
That sounds good. FWIW "Rustacean"=="Rust user" ("Gopher" == "Go user"), but the term is probably still obscure.

What should I use for the 'description'?